nautobot/core/tables.py

```diff
@@ -65,6 +65,8 @@ def __init__(self, *args, user=None, **kwargs):
         # Init table
         super().__init__(*args, **kwargs)
 
+        self.is_filter_applied = is_filter_applied
```

Comment:
can we do something like this here?

```py
self.is_filter_applied = is_filter_applied
self.hide_hierarchy_ui = self.is_filter_applied or self.order_by
```

Then we can just use one argument to the filter?